pkg/enrichment/db/issue.go

```diff
@@ -15,15 +16,15 @@ type issue struct {
 	FalsePositive bool      `db:"false_positive"`
 	UpdatedAt     time.Time `db:"updated_at"`
 
-	Target      string  `db:"target"`
-	Type        string  `db:"type"`
-	Title       string  `db:"title"`
-	Severity    int32   `db:"severity"`
-	CVSS        float64 `db:"cvss"`
-	Confidence  int32   `db:"confidence"`
-	Description string  `db:"description"`
-	Source      string  `db:"source"`
-	CVE         string  `db:"cve"`
+	Target      string  		`db:"target"`
+	Type        string  		`db:"type"`
+	Title       string  		`db:"title"`
+	Severity    int32   		`db:"severity"`
+	CVSS        float64 		`db:"cvss"`
+	Confidence  int32   		`db:"confidence"`
+	Description string  		`db:"description"`
+	Source      string  		`db:"source"`
+	CVE         sql.NullString	`db:"cve"`
```

Comment:
nit: the indentation for the tag strings should be done with spaces, rather than tabs